src/main/java/com/ita/if103java/ims/dao/impl/DashboardDaoImpl.java

```diff
@@ -194,7 +222,7 @@ ORDER BY sum(ts.quantity), name DESC
                  (SELECT w1.id, w2.name, w1.capacity, w1.account_id, w1.top_warehouse_id
                  FROM warehouses w1
                  JOIN warehouses w2
-                 WHERE w1.is_bottom=1 AND w1.active=1 and w2.id=w1.top_warehouse_id) AS cap
+                 WHERE w1.is_bottom=1 AND w2.active=1 and w2.id=w1.top_warehouse_id) AS cap
```

Comment:
Did you change this intentionally?
Just want to be sure that this is not a mistake.